pkg/client/dtclient/client.go

```diff
@@ -316,7 +316,6 @@ func NewPlatformClient(dtURL string, token string, oauthCredentials auth.OauthCr
 
 	classicURL, err := metadata.GetDynatraceClassicURL(oauthClient, dtURL)
 	if err != nil {
-		log.Error("Unable to determine Dynatrace classic environment URL: %v", err)
```

Comment:
I assume this is removed because the error is handled by logging further out as well? So handled twice?
Or is there some other reason to remove logging here? :)

Reply:
yeah it pops up twice